UnityProject/Assets/Scripts/Health/Living/LivingHealthBehaviour.cs

```diff
@@ -231,7 +231,7 @@ public void Extinguish()
 		SyncFireStacks(0);
 	}
 
-	private void SyncFireStacks(float newValue)
+	public void SyncFireStacks(float newValue)
```

Comment:
This is a SyncVar hook, so I'd prefer this to be kept private and instead make a public [Server] annotated method which calls this internally (like [Server] public void ServerSetFireStacks).

When you make SyncVar hook public, it could result in another dev mistakenly calling it from clientside code, which would not work correctly.

Reply:
Thank you for letting me know. Would this work?
```

/// <summary>
/// Sets the serverside fireStacks of a LivingHealthBehaviour to a new value.
/// </summary>
/// <param name="newValue">The new value, clamped to values 0 and above.</param>
[Server]
public void ServerSetFireStacks(float newValue)
{
    this.fireStacks = Math.Max(0, newValue);
}
```